src/test/java/uk/gov/ons/census/casesvc/messaging/FieldCaseUpdatedReceiverIT.java

```diff
@@ -59,9 +60,10 @@ public void setUp() {
     caseRepository.deleteAllInBatch();
   }
 
+  @DirtiesContext(methodMode = AFTER_METHOD)
```

Comment:
I think you meant to put this on the class, not the method?

Reply:
Was just trying to fix Travis didn't work so i'll remove it